web/src/lib/utils/asset-utils.ts

```diff
@@ -189,7 +189,7 @@ export function getAssetRatio(asset: AssetResponseDto) {
 	let width = asset.exifInfo?.exifImageWidth || 235;
 	const orientation = Number(asset.exifInfo?.orientation);
 	if (orientation) {
-		if (orientation == 6 || orientation == -90) {
+		if (orientation == 6 || orientation == -90 || orientation == 8) {
```

Comment:
nit: should we have constants for `6` and `8`? I don't think those values intuitively mean anything to anyone.

Reply:
I would leave digits here. The EXIF tag values for orientation (1-8) are often referred by just numbers